src/models/BaseModel.py

```diff
@@ -38,6 +39,11 @@ def __init__(self, train_size=0.75, test_size=None):
             print('Pre-processed data exists, reading from the file')
             self.cleaned_encoded_data = self.load_cleaned_data()
             print('Data read complete')
+            if force_load_all:
```

Comment:
could you summarize what's happening here? i'm a little lost as to why only `self.description_feature` is re-read.